livecheck/main.py

```diff
@@ -412,6 +412,8 @@ def main(
         chdir(working_dir)
     if debug:
         logging.basicConfig(level=logging.DEBUG)
+    else:
+        logger.configure(handlers=[{"sink": sys.stdout, "level": "INFO"}])
```

Comment:
I think we can just do `logging.basicConfig(level=logging.INFO)` which defaults to stderr. Any reason you want it to be stdout here?

Reply:
you are right, solved